_includes/nav.html

```diff
@@ -71,6 +69,7 @@
   </ul>
 </li>
 <li><a href="/news/">News</a></li>
+<li><a href="/faq/">FAQ</a></li>
```

Comment:
It looks like we don’t have enough space in the header to accommodate the extra link, so FAQ will need to be within the About menu.